### PR TITLE
[MOB-5940] version 6.4.12

### PR DESCRIPTION
**Pull request by roninopf**
## 🔹 Jira Ticket(s)

* [MOB-5940](https://iterable.atlassian.net/browse/MOB-5940)

## ✏️ Description

> Please provide a brief description of what this pull request does.

[MOB-5940]: https://iterable.atlassian.net/browse/MOB-5940?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.4.11
+### Added
+- Added a new function to `IterableAPI` to get the currently set authentication token
```
Comment (Ayyanchira) [inline CHANGELOG.md:7]: A new failure handler which gets called when passed an invalid auth token or null?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+### Changed
+- Changed `setEmail` and `setUserId` to run through the login process if also sent in an authentication token (and a non-`nil` ID)
+- Changed authentication manager behavior to schedule automatic refreshes on JWT retrieval or decoding failures
+
```
Comment (Ayyanchira) [inline CHANGELOG.md:12]: Fixes instances where setting a new authToken for same user failed

**Review by Ayyanchira — COMMENTED** [review]
Added some comments

File: CHANGELOG.md (hunk 1 above)
Comment (roninopf) [reply CHANGELOG.md:7]: OOPS good catch

File: CHANGELOG.md (hunk 2 above)
Comment (roninopf) [reply CHANGELOG.md:12]: OOPS I like this too

**Review by Ayyanchira — APPROVED** [review]
LGTM

**Pull request merged**